bundler/lib/bundler/similarity_detector.rb

```diff
@@ -1,5 +1,8 @@
 # frozen_string_literal: true
 
+# This module is not used anywhere in Bundler
+# It is included for backwards compatibility in-case someone is relying on it
```

Comment:
I left this module in here so I don't pull the rug out from someone who might be using it in their own project. I would like to delete it, is there a deprecation path for stuff like this?